src/components/PeopleSort/PeopleSort.tsx

```diff
@@ -0,0 +1,114 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import { SortBy } from '../../types/SortBy';
+
+export const PeopleSort = () => {
+  const [searchParams] = useSearchParams();
+  const oldSort = searchParams.get('sort');
+  const oldOrder = searchParams.get('order');
+
+  const setIconClass = (columnName: SortBy): string => {
+    if (columnName === oldSort && !oldOrder) {
+      return 'fas fa-sort-up';
+    }
+
+    if (columnName === oldSort && oldOrder) {
+      return 'fas fa-sort-down';
+    }
+
+    return 'fas fa-sort';
+  };
+
+  const addSortToUrlParams = (sort: SortBy): URLSearchParams => {
+    const params = new URLSearchParams(searchParams);
+
+    if (sort) {
+      params.set('sort', sort);
+    } else {
+      params.delete('sort');
+    }
+
+    if (sort === oldSort && !oldOrder) {
+      params.set('order', 'desc');
+    } else {
+      params.delete('order');
```

Comment:
Consider using switch...case here instead

Reply:
I considered using switch...case here, but I feel the code looks too complicated with it. The thing that I should check not only the 'sort' variable, but the oldOrder variable too. Still, I would appreciate it if you give me an example of how it can be done nicely.